### PR TITLE
Added names to all models so as to increase transparency in django-admin

### LINKED ISSUE #599: Add __str__ functions for all models to improve django-admin interface
## Description
None of the models have __str__ functions written. This makes viewing the objects on django-admin, and also debugging more difficult.

## Mocks
admin1
admin2
admin3
admin5

## Acceptance Criteria
- [ ] Add __str__ or __unicode__ functions for all models

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
15-20 mins

@tonythomas01 @tapasweni-pathak Please review this issue

### PR DESCRIPTION
**Pull request by sammy1997**
# Description
Added __unicode__ functions to all models so as to increase transparency in django-admin
Fixes #599 

# Type of Change:
- Code
- Bug fix (non-breaking change which fixes an issue)

# Checklist:
- [x] My PR follows the style guidelines for this project
- [x] Any dependent changes have been merged 
- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by sammy1997** [comment]
@tonythomas01 @tapasweni-pathak  Please review this PR for issue #600

**Comment by sammy1997** [comment]
@tapasweni-pathak Please review this

File: vms/volunteer/models.py [hunk 1]
```diff
@@ -96,3 +96,6 @@ class Volunteer(models.Model):
         blank=True)
 
     user = models.OneToOneField(User)
+
+    def __unicode__(self):
+        return "%s %s" % (self.first_name, self.last_name)
```
Comment (tonythomas01) [inline vms/volunteer/models.py]: For next time, and even this time - its always better to use string formatting (would save time even when we upgrade at some point).

```
return '{0} {1}'.format(self.first_name, ... )
``` 
also, see that `' '` are used

Comment (sammy1997) [reply vms/volunteer/models.py]: Ok @tonythomas01 . I am updating the PR.

**Comment by sammy1997** [comment]
@tonythomas01 Please review this again. I have made the changes

**Comment by sammy1997** [comment]
@tapasweni-pathak @prachi1210 Please review this

**Comment by sammy1997** [comment]
@kamsuri @prachi1210 Please review this.

File: vms/volunteer/models.py [hunk 2]
```diff
@@ -96,5 +96,7 @@ class Volunteer(models.Model):
         blank=True)
 
     user = models.OneToOneField(User)
-    def __str__(self):
```
[the remark this answers is not in the document]
Comment (kamsuri) [reply vms/volunteer/models.py]: Fix this, rest seems fine to me.

[the remark this answers is not in the document]
Comment (sammy1997) [reply vms/volunteer/models.py]: @kamsuri This line was deleted in the commit. I think this is an error from codacy-bot's side. Can you delete this comment?

[the remark this answers is not in the document]
Comment (kamsuri) [reply vms/volunteer/models.py]: It's showing that you have deleted the line but you have left the empty line. Just delete that line.

**Comment by tonythomas01** [comment]
I would not suggest __unicode__ as its dropped in the next version. Look at: https://docs.djangoproject.com/en/1.11/topics/python3/#str-and-unicode-methods 

And probably do something like that ? 

also, use `''` instead, please.

**Pull request merged**